sgl-kernel/csrc/cpu/decode.cpp

```diff
@@ -1134,19 +1445,50 @@ void decode_attention_cpu(
       "decode: expect req_pool_indices to be int64, got ",
       req_pool_indices.scalar_type());
 
+  // check if we have MLA here
+  void* k_buffer_data = k_buffer.data_ptr();
+  void* v_buffer_data = v_buffer.data_ptr();
+  const bool is_mla = (k_buffer_data == v_buffer_data) && (num_heads_kv == 1);
```

Comment:
let's put tighter restrict here:

```suggestion
  const bool is_mla = (k_buffer_data == v_buffer_data) && (num_heads_kv == 1) && (head_size == head_size_v + 64);
```

Reply:
fix gemini code review